tests/test_gw_repeater.sh

```diff
@@ -27,6 +27,35 @@ usage() {
     echo "      --repeater-only - start repeater only"
 }
 
+# This function checks if this script is executed in Microsoft's WSL
+# Docker for Windows currently is unable to forward traffic from the
+# host to the containers. Since we communicate with the UCC listeners
+# on the gateway and the repeaters, expose the UCC listening ports
+# from the containers to the host
+check_wsl() {
+    # Do nothing for non-WSL environments
+    if ! grep -q Microsoft /proc/version; then return; fi
+
+    status "Running in WSL"
+    
+    # Read GW & Repeater UCC ports
+    local GW_UCC_PORT=$(grep ucc_listener_port \
+    ${rootdir}/build/install/config/beerocks_controller.conf | \
+    awk -F'[= ]' '{ print $2 }')
```

Comment:
I know there are already many shellcheck issues in this script, but let's not introduce more
```suggestion
    local GW_UCC_PORT
    GW_UCC_PORT="$(grep ucc_listener_port \
    "${rootdir}/build/install/config/beerocks_controller.conf" | \
    awk -F'[= ]' '{ print $2 }')"
```

Reply:
Done. Also fixed all shellcheck issues while I'm at it.